### PR TITLE
fix insufficient eth label bug

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-2052

## What changed (plus any additional context for devs)

we were not using the currentNetwork but whatever network the app was set to 

## PoW (screenshots / screen recordings)
https://cloud.skylarbarrera.com/Screen-Shot-2021-12-15-22-49-55.60.png

## Dev checklist for QA: what to test
Please test sending on L2s and confirm the txn nonces behave as expected and that the insufficient ETH error doesn't show up

## Final checklist
[x] Assigned individual reviewers?
[x] Added labels?

### CODE REVIEW

File: src/screens/SendSheet.js [hunk 1]
```diff
@@ -608,7 +608,7 @@ export default function SendSheet(props) {
     let label = 'Enter an Amount';
 
     let nativeToken = 'ETH';
-    if (network === networkTypes.polygon) {
+    if (currentNetwork === networkTypes.polygon) {
```
Comment (jinchung) [inline src/screens/SendSheet.js:611]: nice! should we be doing the same thing on line 465?

Comment (skylarbarrera) [reply src/screens/SendSheet.js:611]: spooky, just changed it. nice catch

**Comment by jinchung** [comment]
also line 117 needs to be moved down and also updated to use currentNetwork

**Comment by jinchung** [comment]
> also line 117 needs to be moved down and also updated to use currentNetwork

resolved in my commit

**Review by osdnk — APPROVED** [review]

**Review by jinchung — APPROVED** [review]
🍎

**Pull request merged**